NullPointer error in boolean comparison on reset()

Call reset() without any value for `toAutoUpdate`. App will native crash with NPE.

```
2022-04-24 12:38:03.402 15564-15693/com.maritlabs.topdecked.mtg V/Capacitor/Plugin: To native (Capacitor plugin): callbackId: 70857513, pluginId: CapacitorUpdater, methodName: reset
2022-04-24 12:38:03.402 15564-15693/com.maritlabs.topdecked.mtg V/Capacitor: callback: 70857513, pluginId: CapacitorUpdater, methodName: reset, methodData: {}
2022-04-24 12:38:03.403 15564-15666/com.maritlabs.topdecked.mtg E/Capacitor: Serious error executing plugin
    java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Method.invoke(Native Method)
        at com.getcapacitor.PluginHandle.invoke(PluginHandle.java:121)
        at com.getcapacitor.Bridge.lambda$callPluginMethod$0$com-getcapacitor-Bridge(Bridge.java:592)
        at com.getcapacitor.Bridge$$ExternalSyntheticLambda5.run(Unknown Source:8)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loopOnce(Looper.java:201)
        at android.os.Looper.loop(Looper.java:288)
        at android.os.HandlerThread.run(HandlerThread.java:67)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.Boolean.booleanValue()' on a null object reference
        at ee.forgr.capacitor_updater.CapacitorUpdaterPlugin._reset(CapacitorUpdaterPlugin.java:180)
        at ee.forgr.capacitor_updater.CapacitorUpdaterPlugin.reset(CapacitorUpdaterPlugin.java:196)
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.getcapacitor.PluginHandle.invoke(PluginHandle.java:121) 
        at com.getcapacitor.Bridge.lambda$callPluginMethod$0$com-getcapacitor-Bridge(Bridge.java:592) 
        at com.getcapacitor.Bridge$$ExternalSyntheticLambda5.run(Unknown Source:8) 
        at android.os.Handler.handleCallback(Handler.java:938) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loopOnce(Looper.java:201) 
        at android.os.Looper.loop(Looper.java:288) 
        at android.os.HandlerThread.run(HandlerThread.java:67) 
    
    
    --------- beginning of crash
2022-04-24 12:38:03.405 15564-15666/com.maritlabs.topdecked.mtg E/AndroidRuntime: FATAL EXCEPTION: CapacitorPlugins
    Process: com.maritlabs.topdecked.mtg, PID: 15564
    java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
        at com.getcapacitor.Bridge.lambda$callPluginMethod$0$com-getcapacitor-Bridge(Bridge.java:601)
        at com.getcapacitor.Bridge$$ExternalSyntheticLambda5.run(Unknown Source:8)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loopOnce(Looper.java:201)
        at android.os.Looper.loop(Looper.java:288)
        at android.os.HandlerThread.run(HandlerThread.java:67)
     Caused by: java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Method.invoke(Native Method)
        at com.getcapacitor.PluginHandle.invoke(PluginHandle.java:121)
        at com.getcapacitor.Bridge.lambda$callPluginMethod$0$com-getcapacitor-Bridge(Bridge.java:592)
        at com.getcapacitor.Bridge$$ExternalSyntheticLambda5.run(Unknown Source:8) 
        at android.os.Handler.handleCallback(Handler.java:938) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loopOnce(Looper.java:201) 
        at android.os.Looper.loop(Looper.java:288) 
        at android.os.HandlerThread.run(HandlerThread.java:67) 
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.Boolean.booleanValue()' on a null object reference
        at ee.forgr.capacitor_updater.CapacitorUpdaterPlugin._reset(CapacitorUpdaterPlugin.java:180)
        at ee.forgr.capacitor_updater.CapacitorUpdaterPlugin.reset(CapacitorUpdaterPlugin.java:196)
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.getcapacitor.PluginHandle.invoke(PluginHandle.java:121) 
        at com.getcapacitor.Bridge.lambda$callPluginMethod$0$com-getcapacitor-Bridge(Bridge.java:592) 
        at com.getcapacitor.Bridge$$ExternalSyntheticLambda5.run(Unknown Source:8) 
        at android.os.Handler.handleCallback(Handler.java:938) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loopOnce(Looper.java:201) 
        at android.os.Looper.loop(Looper.java:288) 
        at android.os.HandlerThread.run(HandlerThread.java:67) 
2022-04-24 12:38:03.409 550-564/system_process W/ActivityTaskManager:   Force finishing activity com.maritlabs.topdecked.mtg/.MainActivity
2022-04-24 12:38:03.459 550-4034/system_process I/ActivityManager: Process com.maritlabs.topdecked.mtg (pid 15564) has died: fg  TOP 
```

--- a/android/src/main/java/ee/forgr/capacitor_updater/CapacitorUpdaterPlugin.java
+++ b/android/src/main/java/ee/forgr/capacitor_updater/CapacitorUpdaterPlugin.java
@@ -192,7 +192,7 @@ public class CapacitorUpdaterPlugin extends Plugin implements Application.Activi
 
     @PluginMethod
     public void reset(PluginCall call) {
-        Boolean toAutoUpdate = call.getBoolean("toAutoUpdate");
+        Boolean toAutoUpdate = call.getBoolean("toAutoUpdate", false);
         if (this._reset(toAutoUpdate)) {
             call.resolve();
             return;

